tests/test_code.py

```diff
@@ -4,6 +4,9 @@
 
 
 def test_formatting():
+    if sys.version_info < (3, 8):
+        return
+
```

Comment:
There is a skipif marker from pytest: https://docs.pytest.org/en/latest/how-to/skipping.html#id1

But should we not remove the black unit test, and rather do it as a step in the CI pipeline, like we do in ert?

Reply:
> There is a skipif marker from pytest: https://docs.pytest.org/en/latest/how-to/skipping.html#id1

Thanks, forgot on this one 👍 

> But should we not remove the black unit test, and rather do it as a step in the CI pipeline, like we do in ert?

Can check it in ert ...